application/backup.sh

```diff
@@ -1,4 +1,25 @@
-#!/usr/bin/env sh
+#!/usr/bin/env bash
+
+# Send error to Sentry
+error_to_sentry() {
+    local error_message="$1"
+    local db_name="$2"
+    local status_code="$3"
+
+    if [ ! -z "${SENTRY_DSN}" ]; then
+        curl -X POST "${SENTRY_DSN}" \
```

Comment:
I thought I should check to see if curl is in the Docker image. It is not. But wget is. Here's a [SO question](https://stackoverflow.com/a/6748009) that should help with using wget to make this POST call.

FYI, the way I determined this was to run this command in the repo directory:
```
docker compose run --rm backup bash
```
and then I tried running `curl` and `wget` inside the container. I also noticed by looking at the Dockerfile that `bash` is installed, so that's good for the shebang on line 1.

Reply:
> I thought I should check to see if curl is in the Docker image. It is not. But wget is. Here's a [SO question](https://stackoverflow.com/a/6748009) that should help with using wget to make this POST call.
> 
> FYI, the way I determined this was to run this command in the repo directory:
> 
> ```
> docker compose run --rm backup bash
> ```
> 
> and then I tried running `curl` and `wget` inside the container. I also noticed by looking at the Dockerfile that `bash` is installed, so that's good for the shebang on line 1.

Its true, we don't have curl in the current image. but can't we simply add curl in the Dockerfile ?? i might be wrong here. 

If not i'll need to update like this.  
`if [ ! -z "${SENTRY_DSN}" ]; then
    wget -q --header="Content-Type: application/json" \
         --post-data="{
            \"message\": \"${error_message}\",
            \"level\": \"error\",
            \"extra\": {
                \"database\": \"${db_name}\",
                \"status_code\": \"${status_code}\",
                \"hostname\": \"${HOSTNAME}\"
                }
}" \
         -O - "${SENTRY_DSN}"
fi`

Reply:
We could add curl, but since there's an equivalent solution (wget) without increasing the image size and complexity, let's use do that (use wget).

Reply:
got it